browser/plugins/interaction-breadcrumbs.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
       } catch (e) {
         targetText = '[hidden]'
         targetSelector = '[hidden]'
-        client._logger.error('Cross domain error when tracking click event. See https://docs.bugsnag.com/platforms/browsers/faq/#3-cross-origin-script-errors')
+        client._logger.error('Cross domain error when tracking click event. See https://tinyurl.com/y94fq5zm')
```

Comment:
How many bytes does this save? IMO it's now less clear that this is a docs link.

Reply:
`0.3kB`

Reply:
In that case maybe we could clarify the general format of shortened URLs to something like "See docs: $url"?